src/utilities/https.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+
+export async function getLtr(letterId) {
+  try {
+    const response = await axios.get(
+      `https://rnj9cz9vca.execute-api.us-west-2.amazonaws.com/prod/ltr/${letterId}`
+    );
+    return response;
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+export async function postLtr(body) {
+  try {
+    if (body.questionInputText === '') body.questionInputText = undefined;
+    const response = await axios.post(
+      'https://rnj9cz9vca.execute-api.us-west-2.amazonaws.com/prod/ltr/create',
+      body
+    );
+    return response;
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+export async function editLtr(body) {
+  try {
+    if (body.questionInputText === '') body.questionInputText = undefined;
+    const response = await axios.put(
```

Comment:
the variable name 'response' doesnt tell me anything about what is coming back from this api request